tests/test-a8c-files-utils.php

```diff
@@ -1,6 +1,40 @@
 <?php
 
 class VIP_Go_A8C_Files_Utils_Test extends WP_UnitTestCase {
+	public function get_data_for_normalize_quality() {
+		return [
+			'100_is_forced_lower' => [
+				100,
+				99,
+				true,
+			],
+
+			'below_100_is_fine' => [
+				99,
+				99,
+				false,
+			],
```

Comment:
Can we add some tests for the string version of these values? So if quality were `'100'` or `'99'`. They should _just work_ due to PHP's type juggling, but good to have tests that cover it.